### PR TITLE
libmad: new package at 0.15.1b

### PR DESCRIPTION
**Pull request by alaughlin**
## Description of package
mpeg decoder used by audio applications such as cmus

## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/libmad/build [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -e
+
+for patch in *.patch; do
```
Comment (dylanaraps) [inline community/libmad/build]: Change this entire block to:

```sh
patch -p1 < remove-fforce-mem.patch
```

File: community/libmad/build [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    patch -p1 < "$patch"
+done
+
+./configure \
+    prefix=/usr
```
Comment (dylanaraps) [inline community/libmad/build]: `--prefix=/usr`?

Comment (alaughlin) [reply community/libmad/build]: either seems to work. but I'll change it :)

File: community/libmad/build (hunk 1 above)
Comment (dylanaraps) [reply community/libmad/build]: Actually, you can remove the patch in its entirety and use:

```sh
sed -i'' '/ -fforce-mem/d' configure
```

File: community/libmad/build (hunk 2 above)
Comment (alaughlin) [reply community/libmad/build]: fixed :)

File: community/libmad/build (hunk 1 above)
Comment (alaughlin) [reply community/libmad/build]: good call! fixed :)

**Comment by dylanaraps** [comment]
Looks good. :+1:

**Pull request merged**